### PR TITLE
Add integration test for openapi client generation

### PR DESCRIPTION
**Pull request by aneeshafedo**
Resolves https://github.com/ballerina-platform/openapi-tools/issues/970

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes
 -

### CODE REVIEW

File: ballerina-test/src/test/java/org/ballerinalang/distribution/test/OpenAPIArtifactBuildTest.java [hunk 1]
```diff
@@ -120,6 +120,41 @@ public void buildOpenAPIToBallerinaWithFilterTagsTest() throws IOException,
         }
     }
 
+    @Test(description = "Check openapi to ballerina client generator command", enabled = true)
```
Comment (lnash94) [inline ballerina-test/src/test/java/org/ballerinalang/distribution/test/OpenAPIArtifactBuildTest.java:123]:
```suggestion
    @Test(description = "Check openapi to ballerina client generator command")
```

Comment (aneeshafedo) [reply ballerina-test/src/test/java/org/ballerinalang/distribution/test/OpenAPIArtifactBuildTest.java]: Resolved

**Review by lnash94 — APPROVED** [review]
LGTM

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Pull request merged**